src/js/modules/wallet/controllers/send-scan-qr/send-scan-qr.ctrl.js

```diff
@@ -44,8 +44,29 @@
                         throw new Error("Can't send to Bitcoin Cash address with BTC wallet");
                     } else if (elm.protocol === 'bitcoin:' || elm.protocol === 'bitcoincash:') {
                         $scope.clearRecipient();
-                        $scope.sendInput.recipientAddress = elm.pathname;
-                        $scope.sendInput.recipientDisplay = elm.pathname;
+
+                        switch (elm.protocol) {
+                            case 'bitcoin:':
+                                $scope.sendInput.recipientAddress = elm.pathname;
+                                $scope.sendInput.recipientDisplay = elm.pathname;
+                                break;
+                            case 'bitcoincash:':
+                                // Check if cash address - prepend with 'bitcoincash:' if it's the case
+                                try {
+                                    blocktrailSDK.Wallet.getAddressAndType(elm.protocol + elm.pathname, bitcoinJS.networks.bitcoincash, true);
+                                    $scope.sendInput.recipientAddress = elm.protocol + elm.pathname;
```

Comment:
The result sets a `type` property, which will tell you if it's a cash address or not?